### DISCUSSION TITLE
지하철 구간 추가 기능 질문

### DISCUSSION THREAD

**Discussion by NewWisdom** [woowacourse/java-subway-map-precourse#4] [Q&A] [answered]
안녕하세요 우아한 테크코스 3기 프리코스 참가자입니다. 🏃‍♂️

3차 지하철 노선도 미션 구현 중 
[지하철 구간 추가 기능] 예시에서 이 "순서"의 정의에 대해 궁금증이 생겨 여쭤봅니다.
```
## 노선을 입력하세요.
2호선

## 역이름을 입력하세요.
잠실역

## 순서를 입력하세요.
2

[INFO] 구간이 등록되었습니다.
```
위 입력 상황에서 모든 인덱스는 0부터 시작한다고 하였을 때,
* "순서 = 추가할 구간의 인덱스"일 경우
```
## 지하철 노선도
[INFO] 2호선
[INFO] ---
[INFO] 교대역
[INFO] 강남역
[INFO] 역삼역
[INFO] 잠실역
```
* "순서 = 추가한 역이 가지게 될 인덱스"일 경우
```
## 지하철 노선도
[INFO] 2호선
[INFO] ---
[INFO] 교대역
[INFO] 강남역
[INFO] 잠실역
[INFO] 역삼역
```
미션의 예시에 대해 고민해 보았을 때 이 두가지 정의를 생각하게 되었는데,
미션에서 요구하는 "순서" 의 정의가 있는지, 혹은 프로그램 구현자의 임의로 정의해도 되는지 여쭤봅니다.

감사합니다:)

**Comment by gracefulBrown** [comment, marked as answer]
안녕하세요~
출제 의도는 1부터 순서를 지정하도록 하는것이였습니다!
의도가 조금 더 명확하게 전달되도록 아래와 같이 README를 수정 하였으니 참고해주세요~

...

### 구간 관리
순서는 1부터 시작한다.

...

**Reply by NewWisdom** [reply]
네 확인했습니다! 감사합니다:)
